### DISCUSSION TITLE
helpppp

### DISCUSSION THREAD

**Discussion by ofek8922** [351ELEC/351ELEC#968] [General]
[](url)
#953 #941 #780 
https://github.com/AmberELEC/AmberELEC/assets/144952247/7fdf9610-1014-4196-a64f-c1a99c54ed7d

what i need to do

**Comment by dhwz** [comment]
???

**Reply by ofek8922** [reply]
https://github.com/AmberELEC/AmberELEC/assets/144952247/d44026f8-4d5b-4d91-8af3-2eb3a17b7e60

it keep heppning

**Reply by ofek8922** [reply]
i install a new firmware and its not turning on the device is R35S

**Reply by dhwz** [reply]
This device is not supported

**Reply by ofek8922** [reply]
do you know what firmware is because i tried like 4 and noting
